fastavro/_schema.pyx

```diff
@@ -9,6 +9,7 @@ import json
 from libc.math cimport floor, log10
```

Comment:
I'm not familiar with cython, so I just tried to mirror updates from `_schema_py.py` file